playbook/roles/filebeat/meta/main.yml

```diff
@@ -0,0 +1,57 @@
+galaxy_info:
```

Comment:
Either fill with proper info or remove

Reply:
If we include *.log this will be included automatically

Reply:
Ok, remove.